hs_core/page_processors.py

```diff
@@ -28,8 +28,8 @@ def get_page_context(page, user, resource_edit=False, extended_metadata_layout=N
     file_validation_error = None
     if user.username == 'admin' or \
                     content_model.creator == user or \
-                    user in content_model.owners.all(): # or \
-                    #user in content_model.edit_users.all():
+                    user in content_model.owners.all() or \
+                    user in content_model.edit_users.all():
```

Comment:
if you did `user in (content_models.owners.all() | content_model.edit_users.all())` you'll combine these queries into one.